nemo/collections/nlp/data/machine_translation/machine_translation_dataset.py

```diff
@@ -79,6 +80,7 @@ def __init__(
         cache_data_per_node: bool = False,
         use_cache: bool = False,
         reverse_lang_direction: bool = False,
+        prepend_id: int = None,
```

Comment:
The idea for just calling it prepend_ids was that it can hold anything that we need to prepend. So in the future, in case we wish to prepend anything (which may not necessarily be a language token), we can use the same list.
Though, if you feel prepend_lang_id would be a better name for it, I can go ahead and change.

Reply:
Sounds good.